### PR TITLE
Added resnest-50 model

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

**Review by eaidova — APPROVED** [review]

**Review by eizamaliev — CHANGES_REQUESTED** [review]
Please, add model to models.lst for classification demo.

File: models/public/resnest-50/resnest-50.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Converted model
+
+Image, name - `data`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`
```
Comment (IRDonch) [inline models/public/resnest-50/resnest-50.md:56]:
```suggestion
Channel order is `BGR`.
```

File: models/public/index.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ The models can be downloaded via Model Downloader
 | ResNet 34                   | PyTorch\*                          | [resnet-34-pytorch](./resnet-34-pytorch/resnet-34-pytorch.md) | 73.30%/91.42% | 7.3409 | 21.7892 |
 | ResNet 50                   | PyTorch\*<br>Caffe2\*<br>TensorFlow\*| [resnet-50-pytorch](./resnet-50-pytorch/resnet-50-pytorch.md)<br>[resnet-50-caffe2](./resnet-50-caffe2/resnet-50-caffe2.md)<br>[resnet-50-tf](./resnet-50-tf/resnet-50-tf.md)| 75.168%/92.212%<br>76.128%/92.858%<br>76.38%/93.188%<br>76.17%/92.98% | 6.996~8.216 | 25.53 |
 | ResNet 101                  | Caffe\*                            | [resnet-101](./resnet-101/resnet-101.md) | 76.364%/92.902% | 14.441 | 44.496 |
+| ResNeSt 50                  | PyTorch\*                          | [resnest-50](./resnest-50/resnest-50.md) | 81.11%/95.36% | 10.8148 |  27.4493|
```
Comment (IRDonch) [inline models/public/index.md:54]: I think it should go before the ResNet models (since S comes before T).

**Review by IRDonch — CHANGES_REQUESTED** [review]

**Review by IRDonch — APPROVED** [review]

**Comment by IRDonch** [comment]
@Anna-Mironova There's now a merge conflict; please rebase.

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**